www/components/AgendaIcon.js

```diff
@@ -0,0 +1,17 @@
+import CalendarTodayRoundedIcon from '@material-ui/icons/CalendarTodayRounded';
+import AccessAlarmRoundedIcon from '@material-ui/icons/AccessAlarmRounded';
+import styles from "../styles/AgendaIcon.module.css"
```

Comment:
need a space here to separate between start of a function

Reply:
space where